app/mailers/user_mailer.rb

```diff
@@ -16,25 +16,38 @@ def reservation_status_update(reservation, receipt = false) # rubocop:disable al
     @reservation = reservation
     @status = @reservation.status
 
-    return if !receipt && @status == 'returned overdue' &&
+    return if !receipt && @status == 'returned' && @reservation.overdue &&
               @reservation.equipment_model.late_fee == 0
 
     return if receipt && @reservation.checked_out.nil?
 
+    if @reservation.overdue
+      if @status == 'returned'
+        @status = 'returned overdue'
+      else
+        @status = 'overdue'
+      end
+    end
+
```

Comment:
Isn't that just recreating the old `status` method, essentially? I do agree that we could handle this in the model instead of in the mailer, but if we want a consolidated e-mail method there's going to be a bunch of logic figuring out what kind of e-mail to send.